src/ILCompiler.Compiler/src/Compiler/JitHelper.cs

```diff
@@ -146,6 +146,19 @@ public static void GetEntryPoint(TypeSystemContext context, ReadyToRunHelper id,
                     mangledName = "RhpFltRem";
                     break;
 
+                case ReadyToRunHelper.Mod:
+                    mangledName = "RhpIMod";
```

Comment:
These helpers do not check for overflow/divide-by-zero in CoreRT, but they do check in CoreCLR - it is what RyuJIT expects. This will need more fixes, but this fix should be fine for now.